src/app/(admin)/new_dashboard/guides/components/DeleteEditButtons/index.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
+
+import React from "react";
+import { Box, IconButton } from "@mui/material";
+import ModeEditOutlineOutlinedIcon from "@mui/icons-material/ModeEditOutlineOutlined";
+import DeleteOutlinedIcon from "@mui/icons-material/DeleteOutlined";
+import { useDeleteGuide } from "../../hooks/guideClientHooks";
+import { useParams, useRouter } from "next/navigation";
+
+const DeleteEditButtons: React.FC = () => {
+  const router = useRouter();
+  const params = useParams();
+
+  const id = params?.id ? Number(params.id) : 0;
+  const deleteMutation = useDeleteGuide(id);
+
+  const handleDelete = () => {
+    deleteMutation.mutate();
+  };
+
+  const handleEdit = () => {
+    router.push(`/new_dashboard/guides/edit/${id}`);
+  };
+
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        gap: 0,
+      }}
+    >
+      <IconButton aria-label="edit" onClick={handleEdit}>
+        <ModeEditOutlineOutlinedIcon />
+      </IconButton>
+      <IconButton aria-label="delete" onClick={handleDelete}>
+        <DeleteOutlinedIcon />
+      </IconButton>
+    </Box>
+  );
+};
+
+export default DeleteEditButtons;
```

Comment:
Too specific. What if I want to add more buttons? You can just call it ButtonsGroup or something else.

Also make sure to add a confirmation dialog here when the user wants to delete the guide. Because he might do so by mistake and there's no check here.